ui/page/settings_page.go

```diff
@@ -89,8 +89,9 @@ func NewSettingsPage(l *load.Load) *SettingsPage {
 
 	pg.backButton, pg.infoButton = components.SubpageHeaderButtons(l)
 
-	languagePreference := preference.NewListPreference(pg.WL.Wallet, pg.Theme, languagePreferenceKey,
-		values.DefaultLangauge, values.ArrLanguages).
+	languagePreference := preference.NewListPreference(pg.WL.Wallet, pg.Load,
+		languagePreferenceKey, values.DefaultLangauge, values.ArrLanguages,
+		pg.Icons.ContentClear).
```

Comment:
```suggestion
	languagePreference := preference.NewListPreference(pg.WL.Wallet, pg.Load,
		languagePreferenceKey, values.DefaultLangauge, values.ArrLanguages).
```
since we are passing the load package, no need to pass the icon as it in the load package.